adafruit_rsa/key.py

```diff
@@ -124,12 +128,11 @@ def _assert_format_exists(file_format, methods):
                 "Unsupported format: %r, try one of %s" % (file_format, formats)
             ) from err
 
-    def save_pkcs1(self, format="PEM"):
+    def save_pkcs1(self, format: str = "PEM") -> bytes:
```

Comment:
possibly another spot for the Literal type thing. Although thinking more about it, I'm not sure if that works for strings. Do you happen to know that @tekktrik?

Reply:
Yup!  Literal works for strings!